internal/services/engines/jvm/samples_test.go

```diff
@@ -13,3 +13,46 @@
 // limitations under the License.
 
 package jvm
+
+const (
+	SampleVulnerableHSJVM38 = `
+class T {
+	void f() {
+		String input = "test input";
+		Base64.getEncoder().encodeToString(input.getBytes());
+
+		Base64 base64 = new Base64();
+		String encodedString = new String(base64.encode(input.getBytes()));
+	}
+}
+	`
+
+	SampleVulnerableHSJVM24 = `
+class T {
+	void f(String value) {
+		byte[] decodedValue = Base64.getDecoder().decode(value);
+	}
+}
+	`
+)
+
+const (
+	SampleSafeHSJVM38 = `
+class T {
+	void f() {
+		obj.addContentType("application/x-www-form-urlencoded")
+	}
+}
+	`
+
+	SampleSafeHSJVM24 = `
+class T {
+	void f() {
+		this.decodeSomeRandomValue("value);
+		console.log.println("foo.decode");
+	}
+
+	void decodeSomeRandomValue(String value) {}
+}
+`
+)
```

Comment:
I'm not that creative to write better examples, I welcome suggestions.

Reply:
we could use some examples mentioned on #816 , most of them was on .xml and .gradle files

Reply:
Done.